src/components/content/videos/VideoDetails.js

```
@@ -15,6 +15,11 @@ import useWindowWidth from '@/hooks/useWindowWidth';
 import dynamic from 'next/dynamic';
 import { Toast } from 'primereact/toast';
 import MoreOptionsMenu from '@/components/ui/MoreOptionsMenu';
+import { Buffer } from 'buffer';
+import ZapThreadsWrapper from '@/components/ZapThreadsWrapper';
+import appConfig from '@/config/appConfig';
+import { nip19 } from 'nostr-tools';
+import { Buffer } from 'buffer';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Buffer'. Consider to delete it or rename it.

'Buffer' is defined here:


(lint/suspicious/noRedeclare)